ecs-init/cache/cache_test.go

```diff
@@ -99,6 +102,46 @@ func TestIsAgentCachedTrue(t *testing.T) {
 	}
 }
 
+func TestAgentCacheStatus(t *testing.T) {
+	var cases = []struct {
+		data     string
+		expected CacheStatus
+	}{
+		// Expected states:
+		{"0", StatusUncached},
+		{"1", StatusCached},
+		{"2", StatusReloadNeeded},
+		{"1\n", StatusCached},
+		// Invalid states:
+		{"spurious", StatusUncached},
+		{" ", StatusUncached},
+		{"256", StatusUncached},
```

Comment:
Can you add `{"3", StatusUncached}`?

Reply:
This was intentionally unhandled, the decisions on handling a CacheStatus is on the caller, not this func.